downloadMarcxmlRecords.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from bs4 import BeautifulSoup
+import csv
+import secrets
+import time
+import datetime
+import argparse
+import os
+
+# command line arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--fileName', help='the file of data to be searched. \
+optional - if not provided, the script will ask for input')
+args = parser.parse_args()
+
+if args.fileName:
+    fileName = args.fileName
+else:
+    fileName = input('Enter the file of data to be searched: ')
+
+# run time start, establish variables,
+startTime = time.time()
+
+wskey = secrets.wskey
+baseURL = 'http://www.worldcat.org/webservices/catalog/content/'
+
+with open(fileName) as csvfile:
+    reader = csv.DictReader(csvfile)
+    rowCount = len(list(reader))
```

Comment:
I think you can just move this to line 37, right above the loop

Reply:
Will do